source/faq.txt

```diff
@@ -267,25 +267,40 @@ are some things to check:
 How Can I Prevent a Slow Operation From Delaying Other Operations?
 ------------------------------------------------------------------
 
+You can prevent an operation from slowing down your application by
+specifying the size of the connection pool, which is the cache of
+connections that the driver maintains at any time. By increasing the
+maximum connection pool size, you can reduce application latency.
+
 To control the maximum size of a connection pool, you can set the
 ``maxPoolSize`` option in the :ref:`connection options
 <node-connection-options>`. The default value of ``maxPoolSize`` is
 ``100``. If the number of in-use connections to a server reaches
 ``maxPoolSize``, the next request to that server will wait 
-until a connection becomes available. To prevent long-running operations
-from slowing down your application, you can increase ``maxPoolSize``.
+until a connection becomes available. The following code sets
```

Comment:
Question:
From reading this FAQ entry (especially from the waitQueueTimeout section), I thought that the issue was that the client couldn't open enough sockets/connections to send commands, but in the wording of this sentence, it sounds like the server has to wait because there aren't enough connections/sockets to send the response of a command back to the client. Which one is correct or is it both?

Whatever the answer is, could this relationship be clarified in the introductory paragraph?